src/devices/Ak8963/Ak8963Interface.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Ak8963
+{
+    /// <summary>
+    /// Abstract class for AK8963 the I2C interface. This sensor can be found as a sub 
+    /// I2C sensor like in the MPU9250. The access is done thru another I2C device and the 
+    /// core I2C primitive are different. Use those 3 primitive to define the access to read
+    /// and write bytes to the AK8963
+    /// </summary>
+    public abstract class Ak8963Interface
```

Comment:
I also don't like the name with Interface, and would instead call it something like Base. Also, not sure I understand what the pourpose of having this interface is. Could you explain with a scenario why would this be useful? If there is no good scenario for now for this, I would just instead remove this and the I2c class that implements this, and instead just have the actual device using I2c to communicate

Reply:
I'll rename. No issue but it's really something that does the interface with I2C.